torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -333,50 +352,65 @@ std::unordered_set<const IterDomain*>& ComputeAtRootDomainMap::
   return it->second;
 }
 
+std::unordered_map<IterDomain*, IterDomain*> ComputeAtRootDomainMap::
+    mapBestEffort(
+        const TensorDomain* from_td,
+        const std::vector<IterDomain*>& from_root,
+        const TensorDomain* to_td,
+        const std::vector<IterDomain*>& to_root) const {
+  std::unordered_map<IterDomain*, IterDomain*> id_map;
+  for (auto& from_id : from_root) {
+    bool mapping_found = false;
```

Comment:
What does `mapping_found` do? I understand that it signals a valid map insertion but seems unused in the scope.

Reply:
Thanks. That was just due to copy-pasting. Removed.